src/index.js

```diff
@@ -4,6 +4,10 @@ import './index.scss';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
+if (window.location.protocol !== 'https:') {
+    window.location.replace(`https:${window.location.href.substring(window.location.protocol.length)}`);
```

Comment:
We should just set ‘window.location.protocol = “https:”’ since that will actually trigger a refresh

Reply:
addressed in latest commit!